lib/DDG/Spice/Twitter.pm

```diff
@@ -5,16 +5,23 @@ use strict;
 use DDG::Spice;
 
 spice to => 'https://duckduckgo.com/tw.js?user=$1&callback={{callback}}&current=1';
-triggers query => qr/^(?:twitter\s)?@([a-z0-9_]+)$|^twitter\s([a-z0-9_]+)$/i;
+spice proxy_cache_valid => '200 1h';
+
+my $grammar = qr/(?:on)?(?:at)?/io;
+my $other = qr/(?:user)?(?:account)?(?:id)?/io;
```

Comment:
@nalinbhardwaj these are actually more complex than they need to be. Seeing as we only expect one of the options in each of these regexes we can use an `|`. Seeing as you're making them optional in the triggers, there's no need to make the captures in the regex optional.

```perl
my $grammar = qr/on|at/;
my $other = qr/user|account|id/;
```

Also, you can drop the `/o` modifier as it's not necessary. You can also drop `i` as we're comparing the lowercased query against these, so case insensitive matching isn't needed.